### PR TITLE
Liran's fixes - round 1

### PR DESCRIPTION
**Pull request by alexpirogovski**

### CODE REVIEW

File: charts/mlrun-ce/templates/config/mlrun-env-configmap.yaml [hunk 1]
```diff
@@ -1,17 +1,22 @@
 {{ if .Values.mlrun.enabled}}
+{{- $bucket_name := .Values.global.infrastructure.aws.bucketName | default "mlrun" }}
```
Comment (liranbg) [inline charts/mlrun-ce/templates/config/mlrun-env-configmap.yaml]: u can define that under helpers.tpl and reuse from many places

File: charts/mlrun-ce/templates/config/mlrun-env-configmap.yaml [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
   MLRUN_HTTPDB__PROJECTS__FOLLOWERS: nuclio
   S3_ENDPOINT_URL: {{ include "mlrun-ce.minio.service.url" . }}
+  MLRUN_FUNCTION__SPEC__SERVICE_ACCOUNT__DEFAULT: {{ .Values.mlrun.api.functionSpecServiceAccountDefault }}
   AWS_SECRET_ACCESS_KEY: {{ .Values.minio.rootPassword }}
   AWS_ACCESS_KEY_ID: {{ .Values.minio.rootUser }}
+  MLRUN_STORAGE__AUTO_MOUNT_PARAMS: {{ .Values.mlrun.storageAutoMountParams | default "non_anonymous=True" }}
   MLRUN_HTTPDB__REAL_PATH: s3://
-  MLRUN_ARTIFACT_PATH: s3://mlrun/projects/{{ `{{run.project}}` }}/artifacts
+  MLRUN_ARTIFACT_PATH: s3://{{ $bucket_name }}/projects/{{ `{{run.project}}` }}/artifacts
   MLRUN_FEATURE_STORE__DATA_PREFIXES__DEFAULT: s3://mlrun/projects/{project}/FeatureStore/{name}/{kind}
```
Comment (liranbg) [inline charts/mlrun-ce/templates/config/mlrun-env-configmap.yaml]:
```suggestion
  MLRUN_FEATURE_STORE__DATA_PREFIXES__DEFAULT: s3://{{ bucketname }}/projects/{project}/FeatureStore/{name}/{kind}
```

**Review by liranbg — CHANGES_REQUESTED** [review]

File: charts/mlrun-ce/templates/config/mlrun-env-configmap.yaml (hunk 1 above)
Comment (alexpirogovski) [reply charts/mlrun-ce/templates/config/mlrun-env-configmap.yaml]: Fixed

File: charts/mlrun-ce/templates/config/mlrun-env-configmap.yaml (hunk 2 above)
Comment (alexpirogovski) [reply charts/mlrun-ce/templates/config/mlrun-env-configmap.yaml]: Done

**Pull request merged**